### PR TITLE
Avatar menu

### PR DESCRIPTION
**Pull request by osdnk**
Fixes RNBW-4008

## What changed (plus any additional context for devs)
I added a better menu for the avatar. 

## Screen recordings / screenshots

image

https://streamable.com/p4vrhb

## What to test
See if those buttons are working. MErge after merging the main branch.

No impact on iOS

### CODE REVIEW

**Review by tchayen — APPROVED** [review]
Works

**Comment by tchayen** [comment]
But linter is not pleased

File: src/components/profile/AvatarCircle.js [hunk 1]
```diff
@@ -71,37 +74,62 @@ export default function AvatarCircle({
...
-          <AvatarCircleView backgroundColor={resolvedColor}>
-            <FirstLetter>{accountSymbol}</FirstLetter>
-            {!overlayStyles && <InnerBorder opacity={0.02} radius={65} />}
-          </AvatarCircleView>
-        )}
-      </ShadowStack>
-    </ButtonPressAnimation>
+        <ShadowStack
+          {...position.sizeAsObject(AvatarCircleSize)}
+          backgroundColor={overlayStyles ? 'rgb(51, 54, 59)' : colors.white}
```
Comment (perunt) [inline src/components/profile/AvatarCircle.js:114]: We use that color in two places, maybe it makes sense to move it to colors

**Review by perunt — COMMENTED** [review]
looks good
Should we change the color when switching to dark theme?

**Review by terrysahaidak — CHANGES_REQUESTED** [review]

File: src/components/profile/AvatarCircle.js (hunk 1 above)
Comment (osdnk) [reply src/components/profile/AvatarCircle.js:114]: they ar indeed used in two places, but i don't know what connects those two usages. Probably this is accidental. Do you have any idea how to connect them?

**Review by terrysahaidak — APPROVED** [review]

**Review by perunt — APPROVED** [review]

**Pull request merged**